addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -0,0 +1,573 @@
+            this._onPatched(this.props);
+        });
+        onWillUpdateProps((newProps) => {
+            if (!newProps.readonly && this.state.iframeVisible) {
+                this.state.iframeVisible = false;
+            }
+            if (newProps.value !== this.props.value) {
+                this._resetIframeValue = true;
+            }
+        });
+        onPatched(() => {
+            this._onPatched(this.props);
+        });
+        onWillUnmount(()=> {
+            if (this._qwebPlugin) {
+                this._qwebPlugin.destroy();
+            }
+            if (this.resizerHandleObserver) {
+                this.resizerHandleObserver.disconnect();
+            }
+            this.updateValue();
+        });
+    }
+
+    get currentValue () {
+        return this._textToHtml(this.props.value);
+    }
+    get markupValue () {
+        return markup(this.currentValue || '');
+    }
+    get showIframe () {
+        return this.props.readonly && this.props.cssReadonlyAssetId;
+    }
+    get wysiwygOptions() {
+        return {
+            value: this.props.value,
+            startLater: true,
+            onWysiwygBlur: this._onWysiwygBlur.bind(this),
+            ...this.props.wysiwygOptions,
+            recordInfo: {
+                res_model: this.props.record.resModel,
+                res_id: this.props.record.resId,
+            },
+            collaborationChannel: this.props.isCollaborative && {
+                collaborationModelName: this.props.record.resModel,
+                collaborationFieldName: this.props.fieldName,
+                collaborationResId: parseInt(this.props.record.resId),
+            },
+            mediaModalParams: {
+                ...this.props.mediaModalParams,
+                res_model: this.props.record.resModel,
+                res_id: this.props.record.resId,
+            },
+        };
+    }
+    get translationButtonWrapperStyle() {
+        return `
+            font-size: 15px;
+            position: absolute;
+            right: ${odoo.debug && this.props.codeview ? '40px' : '5px'};
+            top: 5px;
+        `;
+    }
+
+    getValue () {
+        const codeViewEl = this._getCodeViewEl();
+        if (codeViewEl && !codeViewEl.classList.contains('d-none')) {
+            return codeViewEl.value;
+        } else {
+            if (this.wysiwyg) {
+                return this.wysiwyg.getValue();
+            } else {
+                return null;
+            }
+        }
+    }
+     updateValue() {
+        const value = this.getValue();
+        if (value !== null && value !== this.currentValue) {
+            if (this.props.setDirty) {
+                this.props.setDirty(true);
+            }
+            if (this.wysiwyg) {
+                this.wysiwyg.softUpdate = true;
+            }
+            return this.props.update(value);
+        }
+    }
+    async startWysiwyg(wysiwyg) {
+        this.wysiwyg = wysiwyg;
+        await this.wysiwyg.delayedStart();
```

Comment:
I suggest to rename `_start` to something maybe like `startEditing` ? So that here we can do:
```js
this.wysiwyg.startEditing();
```
And in the `start`of wysiwyg you do not have to set `delayedStart`. You just either call `startEditing` directly (legacy), or you don't (owl).